client/dashboard/sites/site-fields/index.tsx

```diff
@@ -247,11 +288,16 @@ export function Status( { site }: { site: Site } ) {
 		);
 	}
 
-	if ( site.launch_status === 'unlaunched' ) {
-		return <SiteLaunchNag site={ site } />;
-	}
-
-	return label;
+	return (
+		<WithHostingFeaturesStatus site={ site }>
```

Comment:
v1 pattern reference:

https://github.com/Automattic/wp-calypso/blob/73a2bde08407def9aa6ad6665061c971003c035e/client/sites/components/sites-dataviews/sites-site-status.tsx#L85

Reply:
I initially opted for this, but then it means the `useInView()` hook will be ALWAYS used in the status column even if the site is not eligible for transfer... not sure if it's performant / good 🤔